From: Kea <isc-kea@packages.debian.org>
Date: Tue, 19 Feb 2019 12:39:35 +0000
Subject: kea_admin_fix

Removed the fallback to the build version of kea-admin since it will not exist on users hosts.
---
 src/bin/admin/kea-admin.in | 2 --
 1 file changed, 2 deletions(-)

--- a/src/bin/admin/kea-admin.in
+++ b/src/bin/admin/kea-admin.in
@@ -50,8 +50,6 @@
 # Include the installed admin-utils.sh if available. Fallback to sources otherwise.
 if test -d "@datarootdir@/@PACKAGE_NAME@"; then
     . "@datarootdir@/@PACKAGE_NAME@/scripts/admin-utils.sh"
-else
-    . "@abs_top_srcdir@/src/bin/admin/admin-utils.sh"
 fi
 
 # Find the installed kea-lfc if available. Fallback to sources otherwise.
